service-tests/distelli.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'distelli', title: 'Distelli' });
+module.exports = t;
+
+t.create('build status')
+  .get('/build/org/application-name.json?token=xxx')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'build',
+    value: Joi.equal(
+      'unknown',
+      'waiting',
+      'queued',
+      'running',
+      'success',
+      'failed',
+      'inaccessible'
+    )
+  }));
```

Comment:
For these tests to be useful, they should hit a working URL, and obtain a response that indicates the service code is working. A deploy that returns failed, running, success, and a build that returns whatever a real build should.